frontend/src/employee-frontend/components/application-page/ApplicationNoteBox.tsx

```diff
@@ -106,15 +111,23 @@ export default React.memo(function ApplicationNoteBox(props: Props) {
     setText(isEdit(props) ? props.note.content : '')
   }, [props])
 
+  const { mutateAsync: updateNote } = useMutationResult(updateApplicationNote)
+  const { mutateAsync: createNote } = useMutationResult(createApplicationNote)
+  const { mutateAsync: deleteNote } = useMutationResult(deleteApplicationNote)
+
   const save = () => {
     if (!isInput(props)) return
 
     setSubmitting(true)
 
     void (
       isEdit(props)
-        ? updateNote(props.note.id, text)
-        : createNote(props.applicationId, text)
+        ? updateNote({
+            applicationId: props.note.applicationId,
+            id: props.note.id,
+            text
+          })
+        : createNote({ applicationId: props.applicationId, text })
```

Comment:
`useMutationResult`'s `mutateAsync` always resolves with a `Success` or `Failure`, it never throws. You should either remove `.catch()` and check for `Failure`, or use plain `useMutation` (I would prefer the latter).